### PR TITLE
fix: importing ethers in test package and build script

### PR DESCRIPTION
**Pull request by fadeev**
* added `"files": ["./hardhat.config.ts"]` to fix importing ethers from hardhat (thanks, @lukema95 for the solution!)
* removed `true` from the build script that made the build script always succeed
* modified the build script to move packages from `dist/packages` to `dist/`

### CODE REVIEW

**Comment by fadeev** [comment]
@andresaiello please, review.

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "npx hardhat clean && npx cpx './packages/tasks/templates/**/*' ./dist/tasks/templates && npx hardhat compile --force && tsc && npx cpx './typechain-types/**/*' ./dist/typechain-types",
+    "build": "npx hardhat clean && npx cpx './packages/tasks/templates/**/*' ./dist/tasks/templates && npx hardhat compile --force && tsc && npx cpx 'dist/packages/*/**' dist && rimraf dist/packages && npx cpx './typechain-types/**/*' ./dist/typechain-types",
```
Comment (GMaiolo) [inline package.json]: I suggest this to be splitted into smaller scripts for readability, i.e.:
```
"copy-templates": "npx cpx './packages/tasks/templates/**/*' ./dist/tasks/templates",
"copy-packages": "npx cpx 'dist/packages/*/**' dist",
"copy-types": "npx cpx './typechain-types/**/*' ./dist/typechain-types",
"build": "npx hardhat clean && npm run copy-templates && npx hardhat compile --force && tsc && npm run copy-packages && rimraf dist/packages && npm run copy-types"
```

**Review by GMaiolo — APPROVED** [review]
utACK
Looks good, left an improvement comment for semantics, but it's not required

File: package.json (hunk 1 above)
Comment (fadeev) [reply package.json]: cf99a19

**Pull request merged**